tests/transforms/test_control.py

```diff
@@ -0,0 +1,192 @@
+from functools import partial
+import numpy as np
+import pennylane as qml
+from pennylane.tape import QuantumTape
+from pennylane.transforms.control import ctrl, ControlledOperation
+from pennylane.tape.tape import expand_tape
+
+
+def assert_equal_operations(ops1, ops2):
+    assert len(ops1) == len(ops2)
+    for op1, op2 in zip(ops1, ops2):
+        assert type(op1) == type(op2)
+        assert op1.wires == op2.wires
+        np.testing.assert_allclose(op1.parameters, op2.parameters)

```

Comment:
Nice :+1:

_Very_ nitpicky, but I wonder if this could be re-written without an explicit for-loop, just so a nice assertion error can be returned showing where in the list the operations differ.

But this is very very tangential, and not needed now.